lib/Doctrine/ORM/Query/ResultSetMapping.php

```diff
@@ -361,9 +364,9 @@ public function addJoinedEntityResult($class, $alias, $parentAlias, $relation)
     /**
      * Adds a scalar result mapping.
      *
-     * @param string $columnName The name of the column in the SQL result set.
-     * @param string $alias      The result alias with which the scalar result should be placed in the result structure.
-     * @param string $type       The column type
+     * @param string     $columnName The name of the column in the SQL result set.
+     * @param string|int $alias      The result alias with which the scalar result should be placed in the result structure.
```

Comment:
I'm widening the type here because we pass an integer in this line:

https://github.com/doctrine/orm/blob/18d6bc375711fd734f6d47b01079725ca7ace966/lib/Doctrine/ORM/Query/SqlWalker.php#L1637

The `$resultAlias` is definitely a string as we can see here:

https://github.com/doctrine/orm/blob/18d6bc375711fd734f6d47b01079725ca7ace966/lib/Doctrine/ORM/Query/SqlWalker.php#L1587

An alternative approach would be to cast the integer to a string.